python/cm/tests/test_api.py

```diff
@@ -0,0 +1,832 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Since this module is beyond QA responsibility we will not fix docstrings here
+# pylint: disable=missing-function-docstring, missing-class-docstring
+
+import os
+import string
+from uuid import uuid4
+
+from django.conf import settings
+from django.db import transaction
+from django.test import TestCase
+from django.urls import reverse
+from rest_framework.test import APIClient
+
+from init_db import init as init_adcm
+from rbac.upgrade.role import init_roles
+
+
+class TestBase(TestCase):
```

Comment:
There is base case test class in `python/adcm/tests/base.py`

Reply:
we decided to leave this testl like that